lib-injection/host_inject.rb

```diff
@@ -0,0 +1,97 @@
+return if ENV["DD_TRACE_SKIP_LIB_INJECTION"] == "true"
+
+Gem.paths = {
+  "GEM_PATH" => "/opt/datadog/apm/library/ruby/#{RUBY_VERSION.split(".")[0..1].join(".")}:/opt/datadog/apm/library/ruby:#{ENV["GEM_PATH"]}"
```

Comment:
What is this parting doing `RUBY_VERSION.split(".")[0..1].join(".")`?

Could you document it or extract it to a variable to make it easier to read?